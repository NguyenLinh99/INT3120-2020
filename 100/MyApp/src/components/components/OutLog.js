import React, { Component } from 'react'
import { TouchableHighlight, View, Text, Alert } from 'react-native';
import { Container, Header, Body, Title, Content } from 'native-base';
export default class OutLog extends Component {
    constructor(props){
        super(props);
    }
    render() {
        return (
            <Container>
                <Header>
                    <Body>
                        <Title>Cá nhân</Title>
                    </Body>
                </Header>
                <Content>
                    <View
                        style={{
                            padding: 8,
                            alignItems: "center",
                            justifyContent: "center",
                            paddingTop: 32,
                        }}
                    >
                        <Text
                        style = {{
                            fontSize: 24,
                            marginBottom: 16,
                        }}
                        >
                            Bạn chưa đăng nhập
                        </Text>
                        <Text
                        style = {{
                            fontSize: 14,
                            marginBottom: 16,
                        }}
                        >
                            Hãy đăng nhập để sử dụng đầy đủ dịch vụ của chúng tôi
                        </Text>
                        <TouchableHighlight
                        style={{
                            backgroundColor: "#0288D1",
                            borderRadius: 8,
                            width: 200,
                            height: 40,
                            justifyContent: "center",
                            alignItems: "center"
                        }}
                        onPress = {this.props.goLogin}
                        >
                            <Text style={{fontSize: 16, color:"white"}}>Đăng nhập</Text>
                        </TouchableHighlight>
                    </View>
                </Content>
            
        </Container>
        )
    }
}
