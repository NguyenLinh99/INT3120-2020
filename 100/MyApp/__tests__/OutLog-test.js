import 'react-native';
import React from 'react';
 
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import OutLog from '../src/components/components/OutLog';
 
describe('Testing', () => {
    it('should match to snapshot', ()=>{
        const snap = renderer.create(<OutLog />);
        expect(snap).toMatchSnapshot();
    });
})
