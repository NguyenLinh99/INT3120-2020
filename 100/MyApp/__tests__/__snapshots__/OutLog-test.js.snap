// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing should match to snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
>
  <View>
    <RCTSafeAreaView
      emulateUnlessSupported={true}
      style={
        Object {
          "backgroundColor": "#F8F8F8",
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#F8F8F8",
            "borderBottomColor": "#a7a6ab",
            "borderBottomWidth": 0.5,
            "elevation": 3,
            "flexDirection": "row",
            "height": 64,
            "justifyContent": "center",
            "left": 0,
            "paddingLeft": 6,
            "paddingRight": 10,
            "paddingTop": 18,
            "right": 0,
            "shadowColor": undefined,
            "shadowOffset": undefined,
            "shadowOpacity": undefined,
            "shadowRadius": undefined,
            "top": 0,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "alignSelf": "center",
              "flex": 1,
            }
          }
        >
          <Text
            numberOfLines={1}
            style={
              Object {
                "color": "#000",
                "fontFamily": "System",
                "fontSize": 17,
                "fontWeight": "700",
                "marginLeft": undefined,
                "paddingLeft": 4,
                "paddingTop": 1,
                "textAlign": "center",
              }
            }
          >
            Cá nhân
          </Text>
        </View>
      </View>
    </RCTSafeAreaView>
  </View>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      contentContainerStyle={
        Array [
          Object {
            "padding": undefined,
          },
          undefined,
        ]
      }
      contentInset={
        Object {
          "bottom": 0,
        }
      }
      enableAutomaticScroll={true}
      enableOnAndroid={false}
      enableResetScrollToCoords={true}
      extraHeight={75}
      extraScrollHeight={0}
      getScrollResponder={[Function]}
      handleOnScroll={[Function]}
      keyboardDismissMode="interactive"
      keyboardOpeningTime={250}
      keyboardShouldPersistTaps="handled"
      keyboardSpace={0}
      onScroll={[Function]}
      resetKeyboardSpace={[Function]}
      resetScrollToCoords={
        Object {
          "x": 0,
          "y": 0,
        }
      }
      scrollEventThrottle={1}
      scrollForExtraHeightOnAndroid={[Function]}
      scrollIntoView={[Function]}
      scrollToEnd={[Function]}
      scrollToFocusedInput={[Function]}
      scrollToPosition={[Function]}
      showsVerticalScrollIndicator={true}
      style={
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
        }
      }
      update={[Function]}
      viewIsInsideTabBar={false}
    >
      <View>
        <View
          style={
            Object {
              "alignItems": "center",
              "justifyContent": "center",
              "padding": 8,
              "paddingTop": 32,
            }
          }
        >
          <Text
            style={
              Object {
                "fontSize": 24,
                "marginBottom": 16,
              }
            }
          >
            Bạn chưa đăng nhập
          </Text>
          <Text
            style={
              Object {
                "fontSize": 14,
                "marginBottom": 16,
              }
            }
          >
            Hãy đăng nhập để sử dụng đầy đủ dịch vụ của chúng tôi
          </Text>
          <View
            accessible={true}
            focusable={false}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#0288D1",
                "borderRadius": 8,
                "height": 40,
                "justifyContent": "center",
                "width": 200,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 16,
                }
              }
            >
              Đăng nhập
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</View>
`;
